frontend/packages/knative-plugin/src/components/overview/__tests__/EventSinkServicesOverviewList.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { ServiceModel } from '@console/knative-plugin';
+import EventSinkServicesOverviewList from '../EventSinkServicesOverviewList';
+import { mockEventSourceData, mockEventSourceDataNoURI } from '../__mocks__/overview-knative-mock';
+
+describe('EventSinkServicesOverviewList', () => {
+  it('should show error info if no sink present or sink,kind is incorrect', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={} />);
+    expect(wrapper.text().includes('No services found for this resource.')).toBe(true);
+  });
+
+  it('should have ResourceLink with proper kind', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={mockEventSourceData} />);
+    const findResourceLink = wrapper.find(ResourceLink);
+    expect(findResourceLink).toHaveLength(1);
+    expect(findResourceLink.at(0).props().kind).toEqual(referenceForModel(ServiceModel));
+  });
+
+  it('should have ResourceLink with proper kind', () => {
```

Comment:
both the test cases have the same description but testing different scenarios

Reply:
done